src/game/utils/flop_helper.rs

```diff
@@ -0,0 +1,261 @@
+use std::collections::HashSet;
```

Comment:
I don't think we need `flop_helper`, right? This stuff is mainly for testing?

Reply:
Wrote it just b/c I found it easier to work with these utilities than listing out individual flops. Figured I could include it if it would make using the aggregate report library easier for others too, but I don't mind removing it if that's not the case.